src/App.js

```diff
@@ -1,17 +1,15 @@
 import React from 'react';
 import './App.css';
+import { TodoList } from './components/TodoList/TodoList';
 
 import users from './api/users';
+import todos from './api/todos';
 
 function App() {
   return (
     <div className="App">
       <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
+      <TodoList users={users} todos={todos} />
```

Comment:
create one array with todo and corresponding user in each element

Reply:
I don't understand why we need one array for todos and users. In "TodoList" we push users in "NewTodo" and must show them in the list, in array where we have todos and users it's hard make. I think we don't need one array